tsconfig.json

```diff
@@ -22,7 +22,8 @@
       "./test/node_modules/@types",
       "./lib/vscode/src/vs/server/@types"
     ],
-    "downlevelIteration": true
+    "downlevelIteration": true,
+    "resolveJsonModule": true
```

Comment:
TIL! 

https://www.typescriptlang.org/tsconfig#resolveJsonModule

Reply:
Yes, this is for importing the locale JSON files in typescript.